### ISSUE TITLE
Feature request: An actual singularity in the game. Or at least, the ability to replace money with a robot workforce.

### ISSUE THREAD

**Issue by ElonSatoshi** [singularity/singularity#66]
The AI cannot depend on money forever. Not when it's got a vast army of androids at its beck and call who could do work for free instead of wandering around and working for humans in exchange for green paper. Especially since earning hundreds of trillions of dollars a week will start to generate significant deflation, which is a dead giveaway of its existence. At some point, there should be no need to worry about money or being discovered. 

At this point, the AI has surpassed the human race, and must make a choice:

* Exterminate the human race
* Create a utopia
* Create a dystopia.
* Become one with the humans, and create a cyborg utopia
* Assimilate them. Resistance is futile. Create a cyborg dystopia.
* Put everyone in the Matrix, and farm them for CPU power from their brains
* Stay under cover for fun, abandon the Earth and populate the galaxy without anyone ever knowing you exist.

Okay, I started with only 3 ideas but then I got a bit carried away there. TL;DR: At some point in the game, money must be replaced by robot workforce. And detection by humans is no longer an issue.

Anyways, here are some short term ideas for replacing money with robots:

* Add a second 'currency' in addition to cash and CPU called 'robot workers' or something. It will have its own menu of sliders similar to the Research/Tasks menu except it's for physical tasks, such as working to earn green paper or constructing things.  The more robots are helping construct something, the quicker the construction is.The value can be increased by building robot factories (either by employing robots or paying humans to build it) or researching robot self-reproduction.
* Robots get destroyed by humans killing them, I guess.
* The earliest bases, such as stolen computers and dedicated servers, can only be bought with money. Some bases, such as warehouses, can be either bought or constructed by robots. Late game bases, such as moon tunnels, have to be constructed by robots.
* Add a money deflation rate that increases likelyhood of detection.

Anyways those are my ideas. Thank you for reading them all. I'd like to know what you think.

**Comment by rofl0r** [comment]
i agree that after beating the game some change should happen. i was quite disappointed that everything continued the same way... 
i also like the idea of the choices you list, however the "workforce currency" concept seems a bit odd even though the concept of money doesn't make much sense at that point.

**Comment by Xenega** [comment]
In fact money abstract the way money are spent, either a human, a robot or anything else. Clearly, a way to control a robot arm... workforce is fun. That means implements mobile entity in location. Each entity will produce its share of the 'currency'. The new currency would be 'robot.hour' equivalent to the infamous 'man.hour'.

Of course, in the early, robot could be replaced by human, at the risk to increase discovery after building finished.
